src/main/java/com/minecolonies/coremod/entity/pathfinding/pathjobs/PathJobFindTree.java

```diff
@@ -84,14 +96,16 @@ public PathJobFindTree(
       final BlockPos home,
       final BlockPos startRestriction,
       final BlockPos endRestriction,
+      final BlockPos furthestRestriction,
       final List<ItemStorage> treesToCut,
       final IColony colony,
       final LivingEntity entity)
     {
-        super(world, startRestriction, endRestriction, new TreePathResult(), entity);
+        super(world, start, startRestriction, endRestriction, (int) Math.sqrt(BlockPosUtil.getDistanceSquared2D(home, furthestRestriction) * 1.5d), AREA_SHRINK, new TreePathResult(), entity);
```

Comment:
there is problem with range calculation when outside the restriction area, made range a ctor param

Reply:
since grow can be negative too? and here it's "shrink", I thought that "grow" is more common name for things like that